### ISSUE TITLE
Comments after configuration (same line) in btrbk.conf

### ISSUE THREAD

**Issue by HaleTom** [digint/btrbk#129]
Would you consider supporting comments after configuration items (on the same line) in `btrbk.conf`?

Or as a work-around, mentioning in the `.example` header comments that a comment needs to be on a line of its own?

**Comment by digint** [comment]
The btrbk man-page mentions:

```
Lines that contain a hash character (#) in the first column are treated as comments.
```

I agree that comments on actual config lines would make sense, as it's "common practice" in most other config files.

**Comment by digint** [comment]
implemented in d31ffb8cb06a87770e174e1f8268721171ba2f40, btrbk-v0.25.0

**Comment by HaleTom** [comment]
Fantastic, thanks!
